answers.txt

```diff
@@ -1,10 +1,12 @@
 1. What is a gitignore file and what should it include for a Java project?
+  a) gitignore file is a text document which contains a list of files that you want git to ignore in your work directory. ".gitignore" tells git which files (or patterns) it should ignore.
+  b) It should include ".mtj.tmp/" ( which is Mobile Tools for Java (J2ME)) for a java project.
```

Comment:
Please include everything that a Java project should include.

Reply:
Sure. Thank you.